### DISCUSSION TITLE
how see tensorboard ??after running this command-!tensorboard --logdir=tts_train_dir?

### DISCUSSION THREAD

**Discussion by omkarade** [coqui-ai/TTS#1747] [General Q&A]
i got this -
![Screenshot (343)](https://user-images.githubusercontent.com/90675376/179421469-a5585ce4-951a-4996-9479-728d20f9b474.png)

**Comment by Ca-ressemble-a-du-fake** [comment]
You've got to open the given address in a browser http://localhost:6006

**Reply by omkarade** [reply]
I try to open. 
This site can’t be reached

**Reply by Ca-ressemble-a-du-fake** [reply]
Localhost is your computer so you tried to open the link from the same OS (if you run it from a virtual machine for example), right ?

**Reply by omkarade** [reply]
I'm using google Colab notebook

**Reply by Ca-ressemble-a-du-fake** [reply]
According to https://colab.research.google.com/github/tensorflow/tensorboard/blob/master/docs/tensorboard_in_notebooks.ipynb#scrollTo=TB0wBWfcVqHz

You can launch it from within the notebook with the following command. Then you may be given a valid url to access to. 
%tensorboard --logdir logs
